### DISCUSSION TITLE
RSV 4180 Compliant CSV parsing

### DISCUSSION THREAD

**Discussion by 1dancook** [andmarti1424/sc-im#644] [Ideas]
In reference to #643 started by @dotancohen 

I think overall it will be good to have such compatibility. 

One related issue is that an RFC 4180 compliant parser may also require RFC 4180 compliant data to parse. That is not always the case with every csv file.

I suppose it could be possible to fall back on the current (or another) parser if there were such an issue. 

Anyway I managed to find this:

https://github.com/birendpatel/CSV-Parser

I don't know if it is completely suitable.

**Comment by dotancohen** [comment]
Yes, I agree that this is an important feature. Especially as the README clearly states:

    CSV / TAB delimited / XLSX file import and export.
